frontend/src/scenes/data-warehouse/settings/source/Logs.tsx

```diff
@@ -0,0 +1,112 @@
+import { LemonButton, LemonCheckbox, LemonInput, LemonTable, LemonTableColumns } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { LOGS_PORTION_LIMIT } from 'lib/constants'
+import { dayjs } from 'lib/dayjs'
+import { pluralize } from 'lib/utils'
+import { LogLevelDisplay } from 'scenes/pipeline/utils'
+
+import { ExternalDataJob, LogEntry } from '~/types'
+
+import { ALL_LOG_LEVELS, schemaLogLogic } from './schemaLogLogic'
+
+const columns: LemonTableColumns<LogEntry> = [
+    {
+        title: 'Timestamp',
+        key: 'timestamp',
+        dataIndex: 'timestamp',
+        width: 1,
+        render: (_, entry) => dayjs(entry.timestamp).format('YYYY-MM-DD HH:mm:ss.SSS UTC'),
+    },
+    {
+        title: 'Level',
+        key: 'level',
+        dataIndex: 'level',
+        width: 1,
+        render: (_, entry) => LogLevelDisplay(entry.level),
+    },
+    {
+        title: 'Run ID',
+        key: 'run_id',
+        dataIndex: 'instance_id',
+        width: 1,
+        render: (_, entry) => entry.instance_id,
+    },
+    {
+        title: 'Message',
+        key: 'message',
+        dataIndex: 'message',
+        width: 6,
+    },
+]
+
+interface LogsTableProps {
+    job: ExternalDataJob
+}
+
+export const LogsView = ({ job }: LogsTableProps): JSX.Element => {
+    const logic = schemaLogLogic({ job })
+    const { logs, logsLoading, logsBackground, isThereMoreToLoad, levelFilters } = useValues(logic)
+    const { revealBackground, loadSchemaLogsMore, setLogLevelFilters, setSearchTerm } = useActions(logic)
+
+    return (
+        <div className="ph-no-capture space-y-2 flex-1">
+            <LemonInput
+                type="search"
+                placeholder="Search for messages containing…"
+                fullWidth
+                onChange={setSearchTerm}
+                allowClear
+            />
+            <div className="flex items-center gap-4">
+                <span>Show logs of type:&nbsp;</span>
+
+                {ALL_LOG_LEVELS.map((type) => {
+                    return (
+                        <LemonCheckbox
+                            key={type}
+                            label={type}
+                            checked={levelFilters.includes(type)}
+                            onChange={(checked) => {
+                                const newLogsTypes = checked
+                                    ? [...levelFilters, type]
+                                    : levelFilters.filter((t) => t != type)
+                                setLogLevelFilters(newLogsTypes)
+                            }}
+                        />
+                    )
+                })}
+            </div>
```

Comment:
I'd arguably say this was too much UI for the small amount of logs. The UI feels a little cluttered and I can't imagine people trying to filter logs so much. I actually really like the logs of Stitch because they're _so_ raw and unedited, makes me trust them.

![image](https://github.com/user-attachments/assets/cd641bed-0e48-45e4-9240-0269ffcf4dce)

Reply:
agreed. I was mirroring our logs in other places. I can remove all of the filters for now